Admin/BaseBlockAdmin.php

```diff
@@ -198,6 +199,24 @@ public function getPersistentParameters()
     /**
      * {@inheritdoc}
      */
+    public function preBatchAction($actionName, ProxyQueryInterface $query, array &$idx, $allElements)
+    {
+        $parent = $this->getParent();
+
+        if ($parent && in_array($actionName, array('delete'))) {
```

Comment:
Why do you use `in_array` when checking for a simple condition?

Reply:
@core23 idk, I think I just had it copy-pasted